app/scripts/lib/verification-methods.js

```diff
@@ -0,0 +1,18 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+/**
+ * List of methods by which a user can verify.
+ * A user can currently only verify via email.
+ * In the future SMS and TOTP could also be included.
+ */
+
+define(function (require, exports, module) {
+  'use strict';
+
+  return {
+    EMAIL: 'email'
+  };
```

Comment:
> Although they're both pretty tiny and they are related and the volume of XHR requests we make for independent JS modules does get on my wick sometimes.

Mine too, though I keep adding files. ;) You know what would be awesome? 2 things:

1. A compilation step that occurs on file change in the background.
2. Using HTTP/2 locally to force push all these files.